30-indoor-air-quality-and-gas-leak/README.md

```diff
@@ -0,0 +1,212 @@
+
+
+### Configuration
+
+The file [`./firmware/sparrow-application/aqi/aqi.cpp`](./firmware/sparrow-application/aqi/aqi.cpp) contains a number of `#define`s that are used to configure how often air quality is measured and the thresholds that trigger an alert:
+
+| Name     | Default  | Unit    | Description |
+|----------|----------|---------|-------------|
+| `AQI_MONITOR_PERIOD` | 60 | seconds | The period in seconds between each air quality reading.
+| `ALERTS_ONLY` | false | boolean | When true, disables reporting of normal air quality and only alerts are sent. |
+| `ALERT_AQI_LEVEL` | 3 | 1-5 | Trigger an alert when the measured AQI is at least this high. (Higher numbers mean poorer air quality.) Set to 0 to disable AQI alerts. |
+| `ALERT_ECO2_LEVEL` | 500 | ppm | Trigger an alert when the measured concentration of CO2 is at or above this level.  Set to 0 to disable CO2 alerts. Note that the concentration of CO2 in the atmosphere is around 420 ppm. |
+| `ALERT_TVOC_LEVEL` | 200 | ppb | Trigger an alert when the measured concentration of VOCs is at or above this level. Set to 0 to disable VOC alerts. |
+
+## Routing Data out of Notehub
+
+Notehub supports forwarding data to a wide range of API endpoints by using the Route feature. This can be used to forward your air quality data to external dashboards and alerts to a realtime notification service.  Here, we will use Twilio SMS API to send a notification of an alert to a phone number.
+
+For an introduction to Twilio SMS routes, please see our [Twilio SMS Guide](https://dev.blues.io/guides-and-tutorials/twilio-sms-guide/).
+
+1. Fill out the required fields for the Twilio SMS route, including "from" and "to" phone numbers, where "from" is your virtual Twilio number, and "to" is the number of the phone that receives the alerts. We will not be using placeholders for these numbers, but will use a placeholder for the message, so set the message field to `[.body.customMessage]`.
+
+2. Under the "Filters" section, set "Notefiles" to "All Notefiles" (the default value.)
+
+3. Under "Data", select "JSONata Expression" and copy and paste the contents of [jsonata/route.jsonata](jsonata/route.jsonata) into the text field "Insert your JSONata expression here".
+
+4. Click "Save Changes".
+
+### Naming Air Quality Monitors
+
+You can optionally create a more recognizable name for the air quality monitors in the project, which is used in SMS alerts. To do this, configure an environment variable that lists the Node IDs and their names:
+
+1. In Notehub, click the Notecard Device corresponding to the gateway.
+2. Click the "Environment" tab.
+3. Add an environment variable with the name `node_names`. The value of the environment variable should follow the format `NodeID:FriendlyName;NoteID2:FriendlyName2;`. For example `203733589236550300220032:Main Office;203733589236550300220033:Workshop`.
```

Comment:
Should we tell them here how to get the node ID? We describe that in the next section, but since we're using it for the first time here, I think it's better here.